.github/workflows/build.yml

```diff
@@ -302,7 +302,7 @@ jobs:
         with:
           version: yarn changeset:version
           publish: yarn changeset:publish
-          commit: "Version Packages with 'latest' tag"
+          commit: "release: release Packages with 'latest' tag"
```

Comment:
Not sure if this causes issues with emojis here, but does this work potentially for these lines (so they're consistent)?

```suggestion
          commit: "📦 release: Release packages with 'latest' tag"
```

Reply:
Looking at the conventional commits spec, commit subjects shouldn't be in sentence case. 👍🏼

Reply:
Ok, so I'd probably make this all lowercase here then:

`release packages with 'latest' tag"`